### ISSUE TITLE
Multiple uses of `test_helper::render_view` lead to crashes in tests.

### ISSUE THREAD

**Issue by Philipp-M** [Philipp-M/trui#13]
I've duplicated the `simple_block_test` test and it crashes with something like:

```
thread 'view::block::tests::simple_block_test' panicked at src/test_helper.rs:50:45:
called `Option::unwrap()` on a `None` value
```

But it seems to be enough to just duplicate:
```rust
let buffer = render_view(
    Size {
        width: 15,
        height: 5,
    },
    sut.clone(),
    AppState,
);
```
in the test to reproduce this.

@zoechi can you take a look at this?

**Comment by zoechi** [comment]
Yes, I'll investigate.

**Comment by zoechi** [comment]
This should be fixed with #12
Some more details there
